lib/v2/xinhua-daily-telegraph/daily.js

```diff
@@ -0,0 +1,71 @@
+const timezone = require('@/utils/timezone');
+const cheerio = require('cheerio');
+const { parseDate, getElementChildrenInnerText, getHTMLWithCache } = require('./utils');
+
+module.exports = async (ctx) => {
+    const date = timezone(new Date(), 8);
+    const dateFormatted = date.getFullYear().toString() + ('0' + (date.getMonth() + 1).toString()).slice(-2) + date.getDate().toString();
+
+    const baseUrl = `http://mrdx.cn/content/${dateFormatted}`;
+    const link = `${baseUrl}/Page01DK.htm`;
+
+    const $ = cheerio.load(await getHTMLWithCache(link, ctx));
```

Comment:
Do not cache this page or the feed won't receive any updates since the cache may keep extending its expiry date.
```suggestion
    const response = await got(link);
    const $ = cheerio.load(response.data);
```